sootup.spark/src/main/java/sootup/spark/PAG.java

```diff
@@ -0,0 +1,45 @@
+package sootup.spark;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.experimental.FieldDefaults;
+import lombok.extern.slf4j.Slf4j;
+import org.jgrapht.Graph;
+import org.jgrapht.graph.DefaultDirectedGraph;
+import org.jgrapht.graph.DefaultEdge;
+import sootup.spark.node.AllocationNode;
+import sootup.spark.node.InstanceFieldRefNode;
+import sootup.spark.node.Node;
+import sootup.spark.node.VariableNode;
+
+/** Pointer assigment graph */
+@Slf4j
+@Getter
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+public class PAG {
+
+  @NonNull Graph<Node, DefaultEdge> delegate = new DefaultDirectedGraph<>(DefaultEdge.class);
```

Comment:
In Call Graph we have a Call Class which is given as type of call. 
It contains the source, target and stmt.
We could do here a similar approach. or it should at least use PAGEdge

Reply:
These are not call edges, rather assignment edges, e.g., between Locals.